elide-async/src/test/java/com/yahoo/elide/async/service/AsyncQueryThreadTest.java

```diff
@@ -69,9 +71,32 @@ public void testProcessQueryGraphQl() throws NoHttpResponseException, URISyntaxE
         queryObj.setQuery(query);
         queryObj.setQueryType(QueryType.GRAPHQL_V1_0);
         when(runner.run(query, user)).thenReturn(response);
-        AsyncQueryThread queryThread = new AsyncQueryThread(queryObj, user, elide, runner, asyncQueryDao, "v1");
+        AsyncQueryThread queryThread = new AsyncQueryThread(queryObj, user, elide, runner, asyncQueryDao, "v1",
+                resultStorageEngine);
         queryResultObj = queryThread.processQuery();
         assertEquals(queryResultObj.getResponseBody(), "ResponseBody");
         assertEquals(queryResultObj.getHttpStatus(), 200);
     }
+
+    @Test
+    public void testConvertJsonToCSV() {
+
+        String csvStr = "[/key][\"value\"]";
+        String jsonStr = "{\"key\":\"value\"}";
+
+        AsyncQuery queryObj = new AsyncQuery();
+        ElideResponse response = new ElideResponse(200, "ResponseBody");
+        String query = "/group?sort=commonName&fields%5Bgroup%5D=commonName,description";
+        String id = "edc4a871-dff2-4054-804e-d80075cf827d";
+        queryObj.setId(id);
+        queryObj.setQuery(query);
+        queryObj.setQueryType(QueryType.JSONAPI_V1_0);
+        when(elide.get(anyString(), any(), any(), anyString())).thenReturn(response);
+        AsyncQueryThread queryThread = new AsyncQueryThread(queryObj, user, elide, runner, asyncQueryDao, "v1",
+                resultStorageEngine);
+
+        String jsonToCSV = queryThread.convertJsonToCSV(jsonStr);
```

Comment:
If the intention is to just test convertJsonToCSV method. You can just create a mock queryThread object and test it. You can remove L-87 to L97